www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/httpserver.xml

```diff
@@ -132,6 +132,28 @@
     <type>checkbox</type>
     <help>If you check this box, a TLS encrypted connection is enforced.</help>
   </field>
+  <field>
+    <id>httpserver.ssl_protocols</id>
+    <label>SSL Protocols</label>
+    <style>selectpicker</style>
+    <type>select_multiple</type>
+    <advanced>true</advanced>
+    <help>Enable specified protocols. Default: TLSv1.2 TLSv1.3.</help>
+  </field> 
+  <field>
+    <id>httpserver.ssl_ciphers</id>
+    <label>SSL Ciphers</label>
+    <type>text</type>
+    <advanced>true</advanced>
+    <help>Specify enabled ciphers in the format understood by the OpenSSL library. Example: ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256. NGINX default (empty field): HIGH:!aNULL:!MD5.</help>
```

Comment:
I would avoid that term since you can switch between OpenSSL and LibreSSL

Reply:
agree. "Specify enabled ciphers in the format understood by the TLS library being used"?

Reply:
:+1:

Reply:
made it somewhat shorter. not bad too i hope